native/lib.rs

```diff
@@ -129,10 +157,23 @@ unsafe impl Sync for Bin {}
 //
 // https://docs.rs/moka/latest/moka/sync/struct.Cache.html#avoiding-to-clone-the-value-at-get
 //
-//                 Key
-//                 |    Value
-//                 |    |
-struct Cream(Cache<Bin, Arc<Bin>>);
+//                 Key             Using byte-sized bounding.
+//                 |    Value      |
+//                 |    |          |
+struct Cream(Cache<Bin, Arc<Bin>>, bool);
```

Comment:
This is fine as-is, and I'm probably overly cautious with tuple types containing more than one slot...

The original tuple struct was just a type alias.  Although it's more effort to convert to a regular (non-tuple) `struct`, it would be less error-prone for future code revisions; e.g., `self.1` ... `self.9`.  Feel free to ignore me here!

Reply:
Nope. You’re correct

Reply:
Note for others: type checking will prevent using the wrong `.N` indexing, but this still should be converted to a regular struct.